FeedAPIChallenge/Feed API/FeedsImageMapper.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  FeedsImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Edil Ashimov on 7/29/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final internal class FeedImageMapper {
```

Comment:
`internal` is a default access level and it can be omitted. Please remove explicit `internal`

```suggestion
final class FeedImageMapper {
```

Reply:
Thanks!Got it!